packages/algolia-search/components/SearchBox/SearchBox.tsx

```diff
@@ -44,6 +44,7 @@ export function SearchBox(props: SearchBoxProps) {
 
   return (
     <TextField
```

Comment:
Is dit voldoende of moeten er nog extra maatregelen genomen worden om hier op te focussen? https://github.com/graphcommerce-org/graphcommerce/blob/7728774cd7e9a4463508a99344b177877e3c826b/packages/magento-search/components/SearchForm/SearchForm.tsx#L26C2-L28

Reply:
Volgens mij is `autoFocus` toevoegen aan het veld voldoende. Heb het bij KommaGo getest, daar werkte het. `inputRef={searchInputElement}` wordt ook gevonden als ik dat log.